irods/path/__init__.py

```diff
@@ -51,22 +49,44 @@ def __new__(cls, *elem_list, **kw):
         absolute_ = kw.pop('absolute',True)
         if kw:
             logging.warning("These iRODSPath options have no effect: %r",kw.items())
-        normalized = cls.resolve_irods_path(*elem_list,**{"absolute":absolute_})
+        normalized = _normalize_iRODS_logical_path(elem_list, absolute_)
         obj = str.__new__(cls,normalized)
         return obj
 
 
-    @staticmethod
-    def resolve_irods_path(*path_elems, **kw):
+def _normalize_iRODS_logical_path(paths, abs_):
+
+    build = []
+
+    if paths and paths[0][:1] == '/':
+        abs_ = True
```

Comment:
Is there a purpose in providing the "absolute" keyword hint if we detect and set it here regardless?

Reply:
It will have the future purpose of allowing relative paths when the absolute keyword is false in the procedure call.
However, if the leading element of the path passed in is '/', then *that* should override.